src/main/kotlin/dsl/Skills.kt

```diff
@@ -0,0 +1,8 @@
+package dsl
+
+
+data class Skills(private val skills: List<Skill> = emptyList()) {
```

Comment:
DSL에 사용되는 스킬 객체의 경우에는 값 객체 + 위임을 통해 구현해봐도 좋을 것 같아요. 
값 객체와 위임을 잘 활용하면  컴파일 단계에서 객체가 아닌 객체가 아닌 타입으로 바꿔줘 성능을 좀 더 챙겨줄 수 있습니다 ㅎ

```suggestion
@JvmInline
value class Skills(private val values: List<Skill> = emptyList()) : List<Skill> by values
```

그리고 객체이름과 변수명이 같도록 설계하는건 지양하는게 좋습니다 ㅎㅎ 왜일까요

Reply:
다시 봐도 동일하게 작성하면 코드의 가독성이 많이 떨어지겠네요! ㅎㅎ skillList로 변경하겠습니다.